src/esn.calendar.libs/app/services/fullcalendar/calendar-configuration.service.spec.js

```diff
@@ -100,7 +105,7 @@ describe('The calFullUiConfiguration service', function() {
           expect(uiConfiguration).to.shallowDeepEqual(CAL_UI_CONFIG);
           expect(esnUserConfigurationService.get).to.have.been.calledOnce;
           expect(esnUserConfigurationService.get).to.have.been.calledWith(moduleConfiguration, moduleName);
-
+          window.$q = $qTemp;
```

Comment:
instead of doing this for every test case, you can use `beforeEach` to setup & `afterEach` for reset

And more, for the thing like this, it deserves an explanation, better if you have a common function with the good name, even jsdoc for it.